lib/opFns.js

```diff
@@ -288,6 +288,18 @@ module.exports = {
       cb(null)
     })
   },
+  EXTCODEHASH: function (address, runState, cb) {
+    var stateManager = runState.stateManager
```

Comment:
Need to add check if it is on constantinople, otherwise throw.

Reply:
👍 (Note to self: check that there are state tests that cover this edge case)